frontend/.env

```diff
@@ -1,4 +1,4 @@
 # Not suggested modifying the backend port
 VITE_BACKEND_URL="http://localhost:8080"
 # Add your google client ID here
-VITE_GOOGLE_CLIENT_ID=""
\ No newline at end of file
+VITE_GOOGLE_CLIENT_ID=14046323103-3qqgan5pco7rrc1nta8ucrnonsgfmmrr.apps.googleusercontent.com
```

Comment:
Please remove your client_ID value do not push the .env values into the PR

Reply:
@AlfiyaSiddique , I will remove it